concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-92	92	breast	1402	BREAST	2
+Marrison-1967-909-93	93	breathe	1407	BREATHE	2
+Marrison-1967-909-94	94	bride	1266	BRIDE	2
+Marrison-1967-909-95	95	bridge	1011	BRIDGE	2
+Marrison-1967-909-96	96	bring	692	BRING	2
+Marrison-1967-909-97	97	brinjal (bengan, eggplant)	1146	AUBERGINE	4
+Marrison-1967-909-98	98	broad	2531	BROAD	4
+Marrison-1967-909-99	99	broom	245	BROOM	2
+Marrison-1967-909-100	100	brother (elder)	1759	OLDER BROTHER	2
+Marrison-1967-909-101	101	brother (younger)	1760	YOUNGER BROTHER	2
+Marrison-1967-909-102	102	bud	1684	BUD	2
+Marrison-1967-909-103	103	buffalo	934	BUFFALO	2
+Marrison-1967-909-104	104	bug	1027	BUG	4
+Marrison-1967-909-105	105	build	1840	BUILD	2
+Marrison-1967-909-106	106	bull	1008	BULL	2
+Marrison-1967-909-107	107	bullet	1114	BULLET	2
+Marrison-1967-909-108	108	burn	2102	BURN	2
+Marrison-1967-909-109	109	bury	1719	BURY	2
+Marrison-1967-909-110	110	bush	1683	BUSH	2
+Marrison-1967-909-111	111	but	1869	BUY	2
+Marrison-1967-909-112	112	butterfly	1791	BUTTERFLY	2
+Marrison-1967-909-113	113	buy	1869	BUY	2
+Marrison-1967-909-114	114	by means of			
+Marrison-1967-909-115	115	calf (of cow)	1989	CALF	4
+Marrison-1967-909-116	116	calf (of leg)	498	CALF OF LEG	2
+Marrison-1967-909-117	117	cane	146	SUGAR CANE	2
+Marrison-1967-909-118	118	carry (generic)	700	CARRY	4
+Marrison-1967-909-119	119	carry (in hand)	412	CARRY IN HAND	4
+Marrison-1967-909-120	120	carry (on back)	465	CARRY ON BACK	2
+Marrison-1967-909-121	121	cat	1208	CAT	2
+Marrison-1967-909-122	122	catch	702	CATCH	2
+Marrison-1967-909-123	123	caterpillar	2247	CATERPILLAR	2
+Marrison-1967-909-124	124	cattle	1153	CATTLE	2
+Marrison-1967-909-125	125	cave	1155	CAVE	2
+Marrison-1967-909-126	126	centipede	1974	CENTIPEDE	2
+Marrison-1967-909-127	127	chaff	2363	CHAFF (HUSK)	4
+Marrison-1967-909-128	128	chair	1476	CHAIR	2
+Marrison-1967-909-129	129	charcoal	1188	CHARCOAL	2
+Marrison-1967-909-130	130	cheap	1887	CHEAP	2
+Marrison-1967-909-131	131	cheek	1730	CHEEK	2
+Marrison-1967-909-132	132	chest	1592	CHEST	2
+Marrison-1967-909-133	133	chicken	1318	CHICKEN	2
+Marrison-1967-909-134	134	chief	301	CHIEFTAIN	2
+Marrison-1967-909-135	135	child	2099	CHILD	2
+Marrison-1967-909-136	136	chillie			
+Marrison-1967-909-137	137	chin	1510	CHIN	2
+Marrison-1967-909-138	138	choose	1750	CHOOSE	2
+Marrison-1967-909-139	139	claw	72	CLAW	2
+Marrison-1967-909-140	140	clay	1399	CLAY	2
+Marrison-1967-909-141	141	clean	704	CLEAN	2
+Marrison-1967-909-142	142	climb	1102	CLIMB	2
+Marrison-1967-909-143	143	clock	1323	CLOCK	2
+Marrison-1967-909-144	144	cloth	1616	CLOTH	2
+Marrison-1967-909-145	145	clothes	1895	CLOTHES	2
+Marrison-1967-909-146	146	cloud	1489	CLOUD	2
+Marrison-1967-909-147	147	coat	603	COAT	2
+Marrison-1967-909-148	148	cock	1511	ROOSTER	2
+Marrison-1967-909-149	149	cold	1287	COLD	2
+Marrison-1967-909-150	150	comb	1475	COMB	2
+Marrison-1967-909-151	151	come	1446	COME	2
+Marrison-1967-909-152	152	cook	1100	COOK (SOMETHING)	2
+Marrison-1967-909-153	153	cook3			
```

Comment:
is "cook3" in teh book?

Reply:
It is not in the orthography file I downloaded, but it shows in the original file before mapping.